packages/cf.js/src/types/structs.ts

```diff
@@ -0,0 +1,28 @@
+import { ABIEncoding, Address } from "./simple-types";
+import { BigNumber } from "ethers/utils";
+
+export interface AppDefinition {
```

Comment:
Structs are typically more than just interfaces, so having interfaces in a structs file seems kinda weird.

Perhaps either just put them in the `src/types` file or in `src/types/index.ts`?

Reply:
I don't like putting real code in index.ts. Prefer index.ts just be an exports aggregator.

Reply:
Yeah me neither actually. Was a poor suggestion.